doc/rtd/reference/yaml_examples/launch_index.rst

```diff
@@ -0,0 +1,19 @@
+.. _cce-launch-index:
```

Comment:
We should document this behavior is EC2-only.

Reply:
Done, and added a link to Amazon's EC2 docs